### PR TITLE
NAS-107352 / 20.10 / filesystem.statfs for SCALE

### PR DESCRIPTION
**Pull request by themylogin**

### CODE REVIEW

File: src/middlewared/middlewared/plugins/filesystem_/stat_linux.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+from middlewared.utils.path import is_child
+
+from .stat_base import FilesystemServiceBase
+
+
+class FilesystemService(Service, FilesystemServiceBase):
+
+    def statfs(self, path):
+        try:
+            st = os.statvfs(path)
```
Comment (william-gr) [inline src/middlewared/middlewared/plugins/filesystem_/stat_linux.py:16]: @anodos325 thought maybe we could use the same for FreeBSD.

Is os.statvfs available in FreeBSD just the same?

For a quick search we are only using it in UI to get free bytes in VM section.

Comment (themylogin) [reply src/middlewared/middlewared/plugins/filesystem_/stat_linux.py:16]: @william-gr yes, it is the same, we will only lose flags and fsid

**Comment by william-gr** [comment]
@anodos325 would like your input here too

**Comment by themylogin** [comment]
@anodos325 ping

**Pull request merged**